lib/ansible/modules/cloud/ovirt/ovirt_vm.py

```diff
@@ -784,6 +784,18 @@
             - NOTE - If there are multiple next run configuration changes on the VM, the first change may get reverted if this option is not passed.
         version_added: "2.8"
         type: bool
+    snapshot:
+        description:
+            - "Snapshot to clone VM from."
+            - "Snapshot with description specified should exist."
+            - "VM having this snapshot should be specified by C(srcvm)."
```

Comment:
Please change to:
`You have to specify C(snapshot_vm) parameter with virtual machine name of this snapshot.`